annotations-runtime/src/test/java/com/github/dynamicextensionsalfresco/webscripts/RequestParamHandler.java

```diff
@@ -53,4 +53,20 @@ public void handleQName(@RequestParam final QName qname) {
 	@Uri("/handleNodeRef")
 	public void handleNodeRef(@RequestParam final NodeRef nodeRef) {
 	}
+
+	@Uri("/handleMissingRequiredParameterDefault")
+	public void handleMissingRequiredParameterDefault(@RequestParam final String expectedCode) {
+	}
+
+	@Uri("/handleMissingRequiredParameter123")
+	public void handleMissingRequiredParameter123(@RequestParam(missingParameterHttpStatusCode = 123) final String expectedCode) {
```

Comment:
This is not really a valid status code (1xx status codes are a bit special), can you replace it with a 4xx or 5xx range status code? This test may break depending on tomcat version.

Reply:
I vote for 418 😊

Reply:
@kerkhofsd overruled by @tgeens 400 :sweat_smile:

Reply:
@kerkhofsd you get the Http Status Code 420 card :sweat: